rocketpy/environment/environment.py

```diff
@@ -1905,6 +1932,14 @@ def process_noaaruc_sounding(self, file):
             outputs="Pressure (Pa)",
             interpolation="linear",
         )
+        # construct barometric height array from pressure array
+        barometric_height_array[:, 0] = 10 * barometric_height_array[:, 0]
```

Comment:
please explain in the code why you're multiplying by 10, it may not be so trivial to realize it.